README.md

```diff
@@ -57,16 +57,15 @@ for more information).
 make               # Build lambdapi.
 make doc           # Build the documentation.
```

Comment:
Remark: there are now two kinds of documentation. make doc generates html pages for the source code and, in docs/, make html generates html pages for the user manual. It would be nice to indicate the two in the top Makefile: make src_doc and make usr_doc?
Btw, I don't like very much that our top Makefile is called GNUmakefile. Is there a good reason for not calling it Makefile?

Reply:
We can rename the user documentation into "manual" as well, if it helps distinguishing the two.

For the makefile, we can write `GNUMakefile` to specify that it uses some GNU features, so that BSD make won't look for it. In our case, the `$(shell ...)` constructs are specific to GNU, so it makes sense. But we can also rename it simply to Makefile, I don't mind; and I guess nobody does.

Reply:
"manual" is too generic? 
"usr_doc" and "dev_doc"?
Renamed "GNUmakefile" by "Makefile" is a good idea :)